bigtable-hbase-parent/bigtable-hbase-integration-tests-2.x/src/test/java/com/google/cloud/bigtable/hbase/test_env/MiniClusterEnv.java

```diff
@@ -68,4 +70,19 @@ public Connection createConnection() throws IOException {
 
     return ConnectionFactory.createConnection(clientConfig);
   }
+
+  @Override
+  public CompletableFuture<AsyncConnection> createAsyncConnection() throws IOException {
```

Comment:
why not return `AsyncConnection` here?

Reply:
This would provide a way to access the future (CF) object if in case we need during tests. I do not yet have a concrete case where this might be required. I am OK to change it to AsyncConnection if you prefer.

Reply:
YAGNI.  I don't ever see a need for a CF.  If we need it, we can add CFs later.